packages/dashboard/src/redux/reducers/trade.js

```diff
@@ -70,6 +74,24 @@ const tradeState = (prevState = initialState, { type, payload }) => {
       }
     }
 
+    case GET_WATCHLIST: {
+      return {
+        ...prevState,
+        isWatchlistLoading: true,
+      }
+    }
+
+    case GET_WATCHLIST_RESPONSE: {
+      return {
+        ...prevState,
+        isWatchlistLoading: false,
+        watchlist: {
+          ...prevState.watchlist,
+          ...payload.data,
```

Comment:
Just making sure you want this to happen.
In your saga you dispatch the response action with the request payload which looks like:
```
{
    "securities": [
        { ... },
        { ... },
        ...
      ],
      total_count: 5
}
```


That means the watchlist object will get set to this data, but in that case you can just overwrite the whole thing.
```suggestion
         return {
          ...prevState,
          isWatchlistLoading: false,
          watchlist: payload.data
        }
```
This suggestion is for the whole returned object